packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -93,7 +94,8 @@ class InkResponse extends StatefulWidget {
     this.borderRadius: BorderRadius.zero,
     this.highlightColor,
     this.splashColor,
-  }) : super(key: key);
+    bool enableFeedback,
+  }) : enableFeedback = enableFeedback ?? true, super(key: key);
```

Comment:
The idea was that this way I don't have to specify the default value in two places (here and below for InkWell). Changed it nonetheless as I didn't have strong feelings.